warc.go

```diff
@@ -95,26 +91,32 @@ func (w *Writer) CloseCompressedWriter() (err error) {
 	return err
 }
 
+func getNextWarcFileName(outputDir, prefix, compression string, serial *atomic.Uint64) (nextFileName string) {
```

Comment:
Same as for generateWarcFileName function name, generateWARCFilename

Reply:
oke